docs/user-guide/sdks-using.md

```diff
@@ -0,0 +1,77 @@
+# Using Zowe SDKs
+
+Learn about using the Zowe Client Software Development Kits (SDKs). The SDKs enable you to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programamatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Node Client SDK
+- Zowe Python Client SDK
+
+## Getting started
+
+To get started with the SDKs, navigate to [Zowe.org Downloads](https://www.zowe.org/download.html) and choose a programming language from the **Zowe Client SDKs** section. The package is downloaded to your computer.
+
+<!--TODO Can we list the artifacts that the user can expect to find in the package after downloading (files, readmes, etc..)? -->
+
+**Example - Node:**
+
+```javascript
+import { Download } from "@zowe/zos-files-for-zowe-sdk";
+
+const session = new Session({
+    hostname: '<host address>',
+    user: '<zosmf user>',
+    password: '<zosmf password>'
+});
+```
```

Comment:
Would be nice to have a brief description here (similar to the python sdk).
Something like: "Create a session object to communicate with z/OS MF" ?